xmppserver/src/main/java/org/jivesoftware/openfire/session/LocalSession.java

```diff
@@ -96,6 +95,12 @@
      */
     private final Map<String, Object> sessionData = new HashMap<>();
 
+    /**
+     * SoftwareVersion (XEP-0092) temporary data. All data stored in this <code>Map</code> disapear when session
```

Comment:
Although this is correct, you're making this comment on a property of the session that disappears. I'd not mention this, as it's confusing. I suggest with replacing this with something like

> Software Version (XEP-0092) data as obtained from the peer on this connection.